evm/src/NttManager/NttManager.sol

```diff
@@ -430,6 +431,11 @@ contract NttManager is INttManager, NttManagerState {
         }
     }
 
+    /// @inheritdoc INttManager
+    function tokenDecimals() public view override(INttManager, RateLimiter) returns (uint8) {
```

Comment:
You could just put this inline in the constructor of NttManagerState to set the immutable